src/MySecondProblematicClass.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+
+class MySecondProblematicClass
+{
+    protected string $test;
+
+    public function test(): string
+    {
+        foreach ($_POST['menu-locations'] as $menu_id) {
+            echo 'ADD warning severity 10 - non-sanitized input variable';
+        }
+        $functionsCookies = $_COOKIE;
```

Comment:
:no_entry_sign: **Error**: Due to server-side caching, server-side based client related logic might not work. We recommend implementing client side logic in JavaScript instead (*WordPressVIPMinimum.Variables.RestrictedVariables.cache_constraints___COOKIE*).